release-1.8/release_notes_draft.md

```diff
@@ -317,69 +318,65 @@ kind.
+- `kubectl proxy` now correctly handles the `exec`, `attach`, and
+  `portforward` commands.  You must pass `--disable-filter` to the command to allow these commands.
+- Added `cronjobs.batch` to "all", so that `kubectl get all` returns them.
+- Added flag `--include-uninitialized` to `kubectl annotate`, `apply`, `edit-last-applied`,
+  `delete`, `describe`, `edit`, `get`, `label,` and `set`. `--include-uninitialized=true` makes
   kubectl commands apply to uninitialized objects, which by default are ignored
-  if the names of the objects are not provided. "--all" also makes kubectl
-  commands apply to uninitialized objects. Please see the
-  [initializer](https://kubernetes.io/docs/admin/extensible-admission-controllers/)
-  doc for more details.
-- Add RBAC reconcile commands through `kubectl auth reconcile -f FILE`. When
+  if the names of the objects are not provided. `--all` also makes kubectl
+  commands apply to uninitialized objects. See the
+  [initializer documentation](https://kubernetes.io/docs/admin/extensible-admission-controllers/) for more details.
+- Added RBAC reconcile commands with `kubectl auth reconcile -f FILE`. When
   passed a file which contains RBAC roles, rolebindings, clusterroles, or
-  clusterrolebindings, it will compute covers and add the missing rules.
-  The logic required to properly "apply" RBAC permissions is more complicated
-  that a json merge since you have to compute logical covers operations between
-  rule sets. This means that we cannot use kubectl apply to update RBAC roles
-  without risking breaking old clients (like controllers).
-- `kubectl delete` no longer scales down workload API objects prior to deletion.
-  Users who depend on ordered termination for the Pods of their StatefulSet’s
-  must use kubectl scale to scale down the StatefulSet prior to deletion.
-- `kubectl run --env` no longer supports CSV parsing. To provide multiple env
-  vars, use the `--env` flag multiple times instead of having env vars separated
-  by commas. E.g. `--env ONE=1 --env TWO=2` instead of `--env ONE=1,TWO=2`.
-- Remove deprecated command `kubectl stop`.
-- Allows kubectl to use http caching mechanism for the OpenAPI schema. The cache
-  directory can be configured through `--cache-dir` command line flag to kubectl.
-  If set to empty string, caching will be disabled.
-- Kubectl performs validation against OpenAPI schema rather than Swagger 1.2. If
+  clusterrolebindings, this command computes covers and adds the missing rules.
+  The logic required to properly apply RBAC permissions is more complicated
+  than a JSON merge because you have to compute logical covers operations between
+  rule sets. This means that we cannot use `kubectl apply` to update RBAC roles
+  without risking breaking old clients, such as controllers.
+- `kubectl delete` no longer scales down workload API objects before deletion.
+  Users who depend on ordered termination for the Pods of their StatefulSets
```

Comment:
second-person instead of third-person - just to be consistent.

Reply:
done